src/coreclr/src/vm/typehandle.cpp

```diff
@@ -468,22 +452,18 @@ BOOL TypeHandle::IsBlittable() const
 {
     LIMITED_METHOD_CONTRACT;
 
+    if (IsArray())
+    {
+        // Single dimentional array's of blittable types are also blittable.
```

Comment:
@jkoritzinsky Is this the semantic? Should 1 dimensional arrays with non-zero lower bounds be considered blittable?

Reply:
Supporting NZLB arrays does seem a bit strange here.

I will consider this resolved though, since there is no change in behavior. 

---
In reply to: [362658073](https://github.com/dotnet/runtime/pull/1201#discussion_r362658073) [](ancestors = 362658073)